test/metabase/sync_database/analyze_test.clj

```diff
@@ -72,3 +78,80 @@
 (expect false (values-are-valid-emails? [100]))
 (expect false (values-are-valid-emails? ["true"]))
 (expect false (values-are-valid-emails? ["false"]))
+
+;; Tests to avoid analyzing hidden tables
+(defn count-un-analyzed-fields [db_id table-name]
+  (let [table-id (db/select-one-id table/Table, :db_id db_id, :active true :name table-name)]
```

Comment:
we're doing :refer [Table] so might as well do `(db/select-one-id Table ...)`

generally that's the pattern we've been using, otherwise aliasing every single model namespace gets out of hand and conflicts with some other stuff sometimes (e.g. there are multiple namespaces that one might alias as "Database")